pkg/codegen/docs/gen.go

```diff
@@ -1130,9 +1138,16 @@ func (mod *modContext) genResource(r *schema.Resource) resourceDocArgs {
 	}
 
 	stateParam := name + "State"
+
+	baseDescription := fmt.Sprintf("Explore the %s resource of the %s module,"+
+		" including examples, input properties, output properties,"+
+		" lookup functions, and supporting types. ", name, mod.mod)
```

Comment:
I suggest removing the space at the end of the sentence here and instead, make the whitespace character between the two descriptions explicit on line 1150. So `baseDescription +" "+ metaDescriptionRegexp.FindString((r.Comment))`. Just so someone doesn't accidentally try to remove the space from this sentence.

Reply:
good thinking